internal/provider/data_source_cloudexport_item_test.go

```diff
@@ -183,3 +221,338 @@ func makeTestCloudExportDataSourceItems(apiURL string) string {
+					Check: resource.ComposeTestCheckFunc(
+						resource.TestCheckResourceAttr(ceIBMDS, "type", "CLOUD_EXPORT_TYPE_KENTIK_MANAGED"),
+						resource.TestCheckResourceAttr(ceIBMDS, "enabled", "true"),
+						resource.TestCheckResourceAttr(ceIBMDS, "name", fmt.Sprintf("%s-ibm-export-item", getAccTestPrefix())),
+						resource.TestCheckResourceAttr(ceIBMDS, "description", fmt.Sprintf("%s-description-ibm", getAccTestPrefix())),
+						resource.TestCheckResourceAttr(ceIBMDS, "plan_id", getKentikPlanIDAccTests()),
+						resource.TestCheckResourceAttr(ceIBMDS, "cloud_provider", "ibm"),
+						resource.TestCheckResourceAttr(
+							ceIBMDS,
+							"ibm.0.bucket",
+							fmt.Sprintf("%s-terraform-ibm-bucket", getAccTestPrefix())),
+						resource.TestCheckResourceAttr(ceIBMDS, "bgp.0.apply_bgp", "true"),
+						resource.TestCheckResourceAttr(
+							ceIBMDS,
+							"bgp.0.use_bgp_device_id",
+							fmt.Sprintf("%s-device-id", getAccTestPrefix())),
+						resource.TestCheckResourceAttr(
+							ceIBMDS,
+							"bgp.0.device_bgp_type",
+							fmt.Sprintf("%s-device-bgp-type", getAccTestPrefix())),
+					),
+				},
+			},
+		})
+	}
+}
+
+func TestAccDataSourceCloudExportItemAzure(t *testing.T) {
+	if skipIfNotAcceptance() {
+		checkRequiredEnvVariables(t)
+		ce, err := createTestAccCloudExportItemAzure()
+		assert.NoError(t, err)
+
+		resource.ParallelTest(t, resource.TestCase{
+			ProviderFactories: providerFactories(),
+			Steps: []resource.TestStep{
+				{
+					Config: makeTestAccCloudExportDataSourceItems(models.CloudProviderAzure, ce),
+					Check: resource.ComposeTestCheckFunc(
+						resource.TestCheckResourceAttr(ceAzureDS, "type", "CLOUD_EXPORT_TYPE_CUSTOMER_MANAGED"),
+						resource.TestCheckResourceAttr(ceAzureDS, "enabled", "true"),
+						resource.TestCheckResourceAttr(ceAzureDS, "name", fmt.Sprintf("%s-azure-export-item", getAccTestPrefix())),
+						resource.TestCheckResourceAttr(ceAzureDS, "description", fmt.Sprintf("%s-description-azure", getAccTestPrefix())),
+						resource.TestCheckResourceAttr(ceAzureDS, "plan_id", getKentikPlanIDAccTests()),
+						resource.TestCheckResourceAttr(ceAzureDS, "cloud_provider", "azure"),
+						resource.TestCheckResourceAttr(ceAzureDS, "azure.0.location", "centralus"),
+						resource.TestCheckResourceAttr(
+							ceAzureDS,
+							"azure.0.resource_group",
+							fmt.Sprintf("%s-traffic-generator", getAccTestPrefix())),
+						resource.TestCheckResourceAttr(ceAzureDS, "azure.0.storage_account", fmt.Sprintf("%s-sa", getAccTestPrefix())),
+						resource.TestCheckResourceAttr(ceAzureDS, "azure.0.subscription_id", fmt.Sprintf("%s-sid", getAccTestPrefix())),
+						resource.TestCheckResourceAttr(ceAzureDS, "azure.0.security_principal_enabled", "true"),
+						resource.TestCheckNoResourceAttr(ceAzureDS, "bgp.0.apply_bgp"),
+					),
+				},
+			},
+		})
+	}
+}
+
+func skipIfNotAcceptance() bool {
+	_, accTest := os.LookupEnv(resource.EnvTfAcc)
+	return accTest
```

Comment:
What about if `TF_ACC` is set, but empty or set to "0"? Does the Terraform SDK count  it as acceptance test run?

A safer solution:
```
val, ok := os.LookupEnv(resource.EnvTfAcc)
if ok {
   i, err = strconv.Atoi(v)
   return !err && i > 0
} else {
  return False
}
```

Reply:
The terraform [documentation](https://www.terraform.io/plugin/sdkv2/testing/acceptance-tests) says that the TF_ACC value can be anything.
_TF_ACC Environment Variable: Set to any value. Prevents developers from incurring unintended charges when running other Go tests._